nl_server/config.py

```diff
@@ -124,12 +147,17 @@ def parse_v1(embeddings_map: Dict[str, any]) -> EmbeddingsConfig:
                                             usage=model_info['usage'],
                                             gcs_folder=model_info['gcs_folder'])
     elif model_type == ModelType.VERTEXAI:
+      if model_name not in vertex_ai_model_map:
```

Comment:
throw exception!?

Reply:
hm, throwing an exception worries me because I've only added the vertex_ai_model map to autopush and dev, so if we deploy this code anywhere else that spins up an nl server, then this exception will get thrown .. 

one thing I could do is check if vertex ai is allowed here before doing this?

Reply:
looks like we need a bit in embeddings.yaml to indicate whether a model should be loaded or not?

Reply:
hm, should this be a bit in embeddings.yaml or instance specific

Reply:
This should be instance specific i think. Different instance (bio, sdg, un, custom dc, dev) will use very different indexes. Maybe add them to the instance yaml?